### PR TITLE
Add support for Slm

### PR DESCRIPTION
**Pull request by venticco**
This implements base support for Slm templates (port of Slim), mentioned by issue [#177].

[Review on Reviewable](https://reviewable.io/reviews/tj/consolidate.js/202)

### CODE REVIEW

**Comment by doowb** [comment]
Thanks! I'll get this merged in soon.

**Comment by venticco** [comment]
What's the status of merge? Can it be expected any soon?

File: lib/consolidate.js [hunk 1]
```diff
@@ -1194,6 +1194,38 @@ exports.react.render = reactRenderer('string');
...
+/**
+ * Slm string support.
+ */
+
+exports.slm.render = function(str, options, fn) {
+  return promisify(fn, function (fn) {
+    var engine = requires.slm;
+    if (!engine) {
+      try {
+        engine = requires.slm = require('slm');
```
Comment (BridgeAR) [inline lib/consolidate.js]: Please write this as:
```js
var engine = requires.slm || (requires.slm = require('slm'));
```

**Comment by BridgeAR** [comment]
LGTM besides my comment. Please rebase and address that. In that case I'll merge it right away.

**Comment by BridgeAR** [comment]
Merged c459107358f969ab38bfde060426749508057d20

**Pull request closed without merging**